docker-compose.localstack.yml

```
@@ -0,0 +1,26 @@
+version: '3.4'
+services:
+  localstack:
+    container_name: local_stack
+    image: localstack/localstack
```

```suggestion
    image: localstack/localstack:3
```
Perhaps better to specify the major version number.